common/web/gesture-recognizer/src/engine/headless/touchpointCoordinator.ts

```diff
@@ -36,59 +34,42 @@ export class TouchpointCoordinator<HoveredItemType> extends EventEmitter<EventMa
     this.inputEngines.push(engine);
   }
 
-  private readonly onNewTrackedPath = (touchpoint: SimpleGestureSource<HoveredItemType>) => {
+  private readonly onNewTrackedPath = async (touchpoint: SimpleGestureSource<HoveredItemType>) => {
```

Comment:
why async now? Can you add a return type (`Promise<void>` I guess?) for clarity?

Reply:
That snuck in from the original changeset I spun this PR off from.  Should definitely revert.

```suggestion
  private readonly onNewTrackedPath = (touchpoint: SimpleGestureSource<HoveredItemType>) => {
```